### PR TITLE
Support `evaluation_metrics_plot` directly from `EvaluationRow`

### PR DESCRIPTION
**Pull request by hannahilea**
...the obvious follow-up for #45. :)

### CODE REVIEW

File: src/plotting.jl [hunk 1]
```diff
@@ -300,9 +300,15 @@ end
...
 """
     evaluation_metrics_plot(data::Dict; resolution=(1000, 1000), textsize=12)
+    evaluation_metrics_plot(row::EvaluationRow; kwargs...)
 
-Plots all evaluation metrics Lighthouse has to offer.
+Plot all evaluation metrics generated via [`evaluation_metrics_row`](@ref) and/or
+[`evaluation_metrics`](@ref) in a single image.
 """
+function evaluation_metrics_plot(row::EvaluationRow; kwargs...)
+    return evaluation_metrics_plot(_observation_table_to_inputs(row); kwargs...)
```
Comment (ericphanson) [inline src/plotting.jl:309]: does this work? I thought it would need to be something like `_evaluation_table_to_inputs`

Comment (hannahilea) [reply src/plotting.jl]: :headdesk: yup, 100%. fixed.

**Review by ericphanson — APPROVED** [review]

**Comment by hannahilea** [comment]
I flipped the main entrypoint to `evaluation_metrics_plot` such that it takes a row (since that then enforces that the function gets all the keys it expects!), hence the rereview request. (A well-tested change, assuming the tests all pass!)

**Review by ericphanson — APPROVED** [review]
I like it even better!

**Pull request merged**